numpy/core/fromnumeric.py

```diff
@@ -52,6 +52,25 @@ def _wrapit(obj, method, *args, **kwds):
     return result
 
 
+def _is_compat_error(e, method):
```

Comment:
Could you add a docstring here explaining what the function does? Usual docstring format would be good. I know that some of the other functions are lacking in complete documentation, but they are old...